content/en/apps/guides/data/users-bulk-load.md

```diff
@@ -4,6 +4,8 @@ linkTitle: "Bulk Load Users"
 weight: 15
 description: >
   Notes for how to bulk load users
+aliases:
```

Comment:
Aside from adding an alias to catch the redirect, I feel like we can jazz things up with a link/read more/what not to:
* [CSV To Docs](https://docs.communityhealthtoolkit.org/apps/guides/data/csv-to-docs/)
* [`cht-conf`](https://github.com/medic/cht-conf) (which is strange we link to the GH repo in a bunch of docs places, but a fix for another day)

Reply:
Good idea. I have made an attempt in https://github.com/medic/cht-docs/pull/607/commits/c394a35240cb894f510724f989208db596382a80 with a page header.

<img width="620" alt="image" src="https://user-images.githubusercontent.com/789512/149402243-197390cf-2e33-41ce-a2fe-bee9f36b9eec.png">